src/domain/valueObject/httpResponseCode.go

```diff
@@ -8,38 +8,27 @@ import (
 	voHelper "github.com/speedianet/os/src/domain/valueObject/helper"
 )
 
-const responseCodeExpression = "^([1-5][0-9][0-9])$"
+const responseCodeRegex = "^([1-5][0-9][0-9])$"
 
 type HttpResponseCode uint64
```

Comment:
ResponseCode may be of type uint16 easily.

Reply:
Done!